Commands/deadchat.py

```
@@ -0,0 +1,130 @@
+        r = proxy_request(
+            "GET",
+            url,
+            params={"jsonBasic": 1}
+        )
+
+        if not r.ok:
+            return []
+
+        try:
+            data = r.json()
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)